ocaml/xapi/xapi_vm.ml

```diff
@@ -61,8 +61,16 @@ let assert_can_boot_here ~__context ~self ~host =
   assert_can_boot_here ~__context ~self ~host ~snapshot ()
 
 let retrieve_wlb_recommendations ~__context ~vm =
+  let open Workload_balancing in
```

Comment:
why did you add this `open`?

Reply:
The syntax `Module.(...)` is not supported in OCaml 4.02.3, so I used a local open instead to avoid repetition.